### ISSUE TITLE
Missing module

### ISSUE THREAD

**Issue by arialcrime** [roboDocs/ScaleFast#1]
Hi,

I just downloaded the most recent version of ScaleFast through Mechanic. When I try to start the extension, I get this traceback in the output window.

```
Traceback (most recent call last):
​  File "scaleFast.py", line 35, in <module>
ModuleNotFoundError: No module named 'mutatorScale.objects'
```

**Comment by gferreira** [comment]
hi, thanks for creating an issue for this.

mutatorScale and the UI components with linked parameters were moved to separate repositories, and are now [included in ScaleFast using git submodules](https://github.com/roboDocs/ScaleFast/tree/master/ScaleFast.roboFontExt/lib). the problem is, the actual submodules are not included when you clone or download the repository… :-/

I think using [git subtree](http://www.atlassian.com/blog/git/alternatives-to-git-submodule-git-subtree) instead of git submodule will fix it. I’ll give it a try soon.

**Comment by gferreira** [comment]
ps. to make it work right now, you‘ll need to download or clone [mutatorScale](http://github.com/roboDocs/dynamicParameters) and [dynamicParamaters]() separately, and copy them into the `lib` folder inside the extension.

**Comment by gferreira** [comment]
this should be fixed in the latest update (v1.0.1) – available on 🔧⚙️

cc @arialcrime @FosterType

**Comment by arialcrime** [comment]
Sorry for the delay. I just did a quick test using Robofont 3.1 and I can confirm the new version solves the problem.
